### PR TITLE
use OCP\EventDispatcher\Event over Symfony's deprecated Event

### PR DESCRIPTION
**Pull request by blizzz**
leaves GenericEvent  to be replaced

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -67,3 +67,7 @@ Options -Indexes
 <IfModule pagespeed_module>
   ModPagespeed Off
 </IfModule>
+#### DO NOT CHANGE ANYTHING ABOVE THIS LINE ####
```
Comment (ChristophWurst) [inline .htaccess]: :roll_eyes:

File: apps/twofactor_backupcodes/tests/Unit/Migration/CheckBackupCodeTest.php [hunk 2]
```diff
@@ -23,15 +23,9 @@
 
 namespace OCA\TwoFactorBackupCodes\Tests\Unit\Migration;
 
-use OCA\TwoFactorBackupCodes\Event\CodesGenerated;
-use OCA\TwoFactorBackupCodes\Listener\RegistryUpdater;
 use OCA\TwoFactorBackupCodes\Migration\CheckBackupCodes;
-use OCA\TwoFactorBackupCodes\Provider\BackupCodesProvider;
```
Comment (ChristophWurst) [inline apps/twofactor_backupcodes/tests/Unit/Migration/CheckBackupCodeTest.php]: 🧹

**Review by ChristophWurst — APPROVED** [review]
:rocket:

File: apps/twofactor_backupcodes/tests/Unit/Migration/CheckBackupCodeTest.php (hunk 2 above)
Comment (blizzz) [reply apps/twofactor_backupcodes/tests/Unit/Migration/CheckBackupCodeTest.php]: it is unused, therefore removed. Or what do you mean?

Comment (ChristophWurst) [reply apps/twofactor_backupcodes/tests/Unit/Migration/CheckBackupCodeTest.php]: This is my *nice cleanup* emoji :)

**Comment by blizzz** [comment]
@juliushaertl could you deign me a review maybe?

**Review by juliusknorr — APPROVED** [review]
:tada:

**Pull request merged**